app/src/main/java/org/mozilla/rocket/content/travel/ui/TravelCityViewModel.kt

```diff
@@ -115,7 +117,13 @@ class TravelCityViewModel(
                     )
                 }
 
-                val wikiResult = getWiki(Uri.encode(englishName))
+                var wikiName = name
+                val wikiNameResult = getCityWikiName(Uri.encode(name))
```

Comment:
We should encapsulate this logic into use case layer, so view model doesn't have to know that it should get the normalized name first then query the extract and image.